docs/pages/reference/config.mdx

```diff
@@ -437,22 +437,22 @@ auth_service:
     # (https://goteleport.com/docs/architecture/proxy/#recording-proxy-mode).
     proxy_checks_host_keys: yes
 
-    # Determines if SSH sessions to cluster nodes are forcefully terminated
-    # after no activity from a client (idle client).
+    # Determines if Server and Database sessions to cluster nodes are forcefully
```

Comment:
This is also valid for app access and desktop access.

Reply:
@zmb3 should we just say "sessions", or is that too ambiguous since it doesn't cover kube access?

Reply:
I think sessions is fine.